app/models/user_column.rb

```diff
@@ -0,0 +1,13 @@
+class UserColumn < ApplicationRecord
+  include Authorizable
```

Comment:
I am not sure this needs to be Authorizable, all users be able to have access to their own columns and we don't really check the permission anywhere other then for controller access.